odoo/addons/base/migrations/12.0.1.3/openupgrade_general_log.txt

```diff
@@ -166,7 +166,7 @@ new model website.sale.payment.acquirer.onboarding.wizard [module website_sale]
 # Found in other module: 156
 # Found with different type: 2
 # In obsolete models: 214
-# New columns: 1369
+# New columns: 1370
```

Comment:
Which is the new column?

Reply:
I didn't updated this. I haven't done a new analysis. Just updated manually the previous. I think it's not important this number. Later, in a future analysis, this will be changed automatically.